djangocms_blog/models.py

```diff
@@ -225,7 +225,7 @@ class Post(KnockerModel, BlogMetaMixin, TranslatableModel):
 
     translations = TranslatedFields(
         title=models.CharField(_('title'), max_length=767),
-        slug=models.SlugField(_('slug'), max_length=767, blank=True,
+        slug=models.SlugField(_('slug'), max_length=752, blank=True,
```

Comment:
`title` field must also be reduced, or the slug truncated when generated, otherwise you may get errors when saving a name longer than the allowed slug field length